daemon/volumes.go

```diff
@@ -103,6 +103,9 @@ func (daemon *Daemon) registerMountPoints(container *container.Container, hostCo
 		}
 
 		for _, m := range c.MountPoints {
+			if _, ok := mountPoints[m.Destination]; ok {
+				return fmt.Errorf("Duplicate mount point '%s'", m.Destination)
```

Comment:
This is really a behavior change.
I think this will end up breaking people who have already brought in a volume and then also do `--volumes-from` on a container with the same volume.